### ISSUE TITLE
Here for help

### ISSUE THREAD

**Issue by SherlockOy** [neokree/MaterialNavigationDrawer#159]
Hi neokree,
I'm a green hand to android develop and I am so admire your work, it looks sooooooo great and very impressive.
Frankly I came here to ask for help, I want to implement the Navigation Drawer just like what you have already done, but I just cannot make the item in different size. For instance, if I set the first item of the drawer to be a picture with the size of 240 dp width, 100 dp height, and the other items will just become exactly the same height as width(wrap_content) x 100 dp.
No matter how I set the property of the other items, they just be the same height as the first drawer item
This is my drawer_list_item.xml
https://github.com/SherlockOy/MyFirstApp/blob/master/app/src/main/res/layout/drawer_list_item.xml
I would be really appreciate if you would help me to find out what on earth is wrong with my code.

Thank you very much !
Sherlock.Oy

**Comment by neokree** [comment]
Well, I set the drawer width and height after the creation of the view using the LayoutParams.
```java
View view = getTheView();
LinearLayout.LayoutParams params = new LinearLayout.LayoutParams(ViewGroup.LayoutParams.MATCH_PARENT,(int)(48 * density));
myLayout.addView(view,params);
```
This code add a view element to the layout with a width of match_parent and an height of 48 dp.
For the density resource you can get using <code>getResources().getDisplayMetrics().density;</code>

**Comment by SherlockOy** [comment]
Thank you for your reply, with your help I final find where I made the mistake. 
I use visibility of layouts to control the appearance, the problem is once I set the layout INVISIBLE, they are just not gone, the solution is replace INVISIBLE with GONE, so the invisible part will not take any space.
Thank you for enlightening me, and I find your way to implement the drawer is much easier to understand, I will try it some day. 
Still thank you sooooooo much!
